.ci/scripts/build_ucc.sh

```diff
@@ -8,7 +8,8 @@ cd "${UCC_SRC_DIR}"
 mkdir -p "${UCC_SRC_DIR}/build"
 cd "${UCC_SRC_DIR}/build"
 "${UCC_SRC_DIR}/configure" --with-ucx="${UCX_INSTALL_DIR}" --with-cuda="${CUDA_HOME}" \
-    --prefix="${UCC_INSTALL_DIR}" --enable-gtest --with-mpi
```

Comment:
Don't forget to return the `--enable-gtest` flag

Reply:
adde back the gtest